docs/Account/integration.mdx

```diff
@@ -48,86 +48,83 @@ pnpm add @biconomy/account
 
 ## Integration
 
-Specify `PRIVATE_KEY`, `BUNDLER_URL`, and `RPC_URL` (for ethers) to create a smart account. 
+Specify `PRIVATE_KEY`, `BUNDLER_URL`, and `RPC_URL` (for ethers) to create a smart account.
 
 <Tabs>
 <TabItem value="js" label="Viem">
 
 ```typescript
-  import { Hex, createWalletClient, http } from "viem";
-  import { privateKeyToAccount } from "viem/accounts";
-  import { polygonMumbai } from "viem/chains";
-  import { createSmartAccountClient } from "@biconomy/account";
-
-  // ----- 1. Generate EOA from private key
-  const account = privateKeyToAccount("PRIVATE_KEY"); 
-  const client = createWalletClient({
-    account,
-    chain: polygonMumbai,
-    transport: http(),
-  });
-  const eoa = client.account.address;
-  console.log(`EOA address: ${eoa}`);
-
-  // ------ 2. Create biconomy smart account instance
-  const smartAccount = await createSmartAccountClient({
-    signer: client,
-    bundlerUrl: "BUNDLER_URL",
-    biconomyPaymasterApiKey: "", // optional
-  });
-  const smartAccountAddress = await smartAccount.getAccountAddress();
-  console.log("Smart Account Address", smartAccountAddress);
-
+import { Hex, createWalletClient, http } from "viem";
+import { privateKeyToAccount } from "viem/accounts";
+import { polygonMumbai } from "viem/chains";
+import { createSmartAccountClient } from "@biconomy/account";
+
+// ----- 1. Generate EOA from private key
+const account = privateKeyToAccount("0x" + "PRIVATE_KEY");
+const client = createWalletClient({
+  account,
+  chain: polygonMumbai,
+  transport: http(),
+});
+const eoa = client.account.address;
+console.log(`EOA address: ${eoa}`);
+
+// ------ 2. Create biconomy smart account instance
+const smartAccount = await createSmartAccountClient({
+  signer: client,
+  bundlerUrl: "https://docs.biconomy.io/dashboard#bundler-url", // <-- Read about this here
```

Comment:
can we please add it as a link to here. since we pass bundler url here. can we keep it consistent, how user links are redirected.